tests/test_json.py

```diff
@@ -333,7 +334,7 @@ def test_config() -> None:
                 }
             ],
             "SourceDateEpoch": 12345,
-            "SplitArtifacts": true,
+            "SplitArtifacts": 3,
```

Comment:
We should really serialize this as a list of strings, this is not exactly usable for users doing something with the JSON output